pyrealm/splash/splash.py

```diff
@@ -88,6 +95,10 @@ def __init__(
         self.kWm = kWm
         """The maximum soil water capacity for sites."""
```

Comment:
There's an error in my implementation - the attributes should be typed, as noted here:
https://github.com/ImperialCollegeLondon/pyrealm/pull/172#discussion_r1466147277

Reply:
Fixed in the latest commit